test/lib/ansible_test/_data/sanity/yamllint/yamllinter.py

```diff
@@ -79,17 +112,40 @@ def check_module(self, conf, path, contents):
         docs = self.get_module_docs(path, contents)
 
         for key, value in docs.items():
-            yaml = value['yaml']
+            yaml_data = value['yaml']
             lineno = value['lineno']
 
-            if yaml.startswith('\n'):
-                yaml = yaml[1:]
+            if yaml_data.startswith('\n'):
+                yaml_data = yaml_data[1:]
                 lineno += 1
 
-            messages = list(linter.run(yaml, conf, path))
+            self.check_parsable(path, yaml_data)
+
+            messages = list(linter.run(yaml_data, conf, path))
 
             self.messages += [self.result_to_message(r, path, lineno - 1, key) for r in messages]
 
+    def check_parsable(self, path, contents):
+        """
+        :type path: str
+        :type contents: str
+        """
+        try:
+            yaml.load(contents, Loader=TestLoader)
+        except MarkedYAMLError as e:
+            # Question for mattclay: If we could use AnsibleLoader, that would take care of
```

Comment:
Now that we have TestConstructor this comment is obsolete isn't it?

Reply:
Yep.

Reply:
Removed.